src/5.0_to_5.1/__test_resources__/contentServer.ts

```diff
@@ -299,56 +299,6 @@ export const contentServer: ContentRecord = {
                     },
                   },
                 },
-                "2sp": {
```

Comment:
This is a folder with a single calculated measure in it.
But there was no entry in `ui/calculated_measures/content` matching this calculated measure (id `5m9`).
So I am simply removing it.